atlasdb-autobatch/src/main/java/com/palantir/atlasdb/autobatch/DisruptorAutobatcher.java

```diff
@@ -63,42 +65,48 @@ private static ThreadFactory createThreadFactory(String safeLoggablePurpose) {
                 .build();
     }
 
-    private final Disruptor<DefaultBatchElement<T, R>> disruptor;
```

Comment:
I thought their claim was that they put up some really good perf numbers in their benchmarking, though I read the ticket and the code you linked, so I'm convinced it's worth trying this out.